tools/proofers/spellcheck_text.inc

```diff
@@ -62,17 +65,17 @@ function spellcheck_text($orig_text, $projectid, $imagefile, $aux_language, $acc
     $returnString .= "<style>" .
                    "  pre { line-height: 2.7; }".
                    "  .proofingfont { font-family: $font_family; $font_size }" .
-                   "  .hl { background-color: yellow; color: black; }" .
+                   "  .hl-punc { background-color: yellow; color: black; }" .
+                   "  .hl-script-0 { background-color: lightgreen; color: black; }" .
```

Comment:
These three new highlighting colors are random placeholders and we need to select at least 4, probably 5 or 6, highlighting colors that will work well before this merges in.

Reply:
Could you instead use just one colour and add a tooltip to indicate what script the offending character belongs to?

Reply:
I don't think that would work visually if a word, for instance, included both Greek and Cyrillic characters.

Reply:
That's true. If a greek word contained one Cyrillic character it wouldn't be obvious anything was wrong.

Reply:
Unless you used two colours and an algorithm so that adjacent characters in different scripts were in different colours. But this would mean there was no relation between the colour and the script.